src/transformers/models/character_bert/modeling_character_bert.py

```diff
@@ -0,0 +1,1938 @@
+# coding=utf-8
+# Copyright Hicham EL BOUKKOURI, Olivier FERRET, Thomas LAVERGNE, Hiroshi NOJI,
+# Pierre ZWEIGENBAUM, Junichi TSUJII, The HuggingFace Inc. and AllenNLP teams.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch CharacterBERT model: this is a variant of BERT that uses the
+CharacterCNN module from ELMo instead of a WordPiece embedding matrix.
+See: “CharacterBERT: Reconciling ELMo and BERT for Word-Level Open-Vocabulary
+Representations From Characters“ https://www.aclweb.org/anthology/2020.coling-main.609/ """
```

Comment:
```suggestion
"""
PyTorch CharacterBERT model: this is a variant of BERT that uses the
CharacterCNN module from ELMo instead of a WordPiece embedding matrix.
See: “CharacterBERT: Reconciling ELMo and BERT for Word-Level Open-Vocabulary
Representations From Characters“ https://www.aclweb.org/anthology/2020.coling-main.609/
"""

```